podcasts/SceneHelper.swift

```diff
@@ -38,6 +48,19 @@ class SceneHelper {
         return appDelegate.window?.rootViewController
     }
 
+    class func getPresentedViewController(base: UIViewController?) -> UIViewController? {
+            guard UIApplication.shared.applicationState == .active else {
+                return nil
+            }
+
+            if let nav = base as? UINavigationController {
+                return getPresentedViewController(base: nav.visibleViewController)
+            } else if let presented = base?.presentedViewController {
+                return getPresentedViewController(base: presented)
+            }
+            return base
+        }
```

Comment:
Done in 9cf0e5e17ae961f5877590d3e490864db41430cf

There's a slight difference between them:

1. This one returns the top most presented view controller, so if there's nothing presented it will return the `MainTabBarController`
2. On the other hand, the one on `AnalyticsCoordinator` iterates over the presented tabs tor return the presented VC